File: bjj-be/src/main/java/com/jipderksen/bjj/models/Training.java
package com.jipderksen.bjj.models;

import lombok.AllArgsConstructor;
import lombok.Data;

import java.util.Date;

/**
 * This class <description of functionality>
 *
 * @author jipderksen
 */
@Data
@AllArgsConstructor
public class Training {
    private Date date;
    private int duration;
    private int numberOfRounds;
    private String technique;
    private String notes;
}

File: bjj-fe/src/app/models/training.ts
export class Training {
  date: Date;
  duration: number;
  numberOfRounds: number;
  technique: string;
  notes: string;

  constructor(date: Date, duration: number, numberOfRounds: number, technique: string, notes: string) {
    this.date = date;
    this.duration = duration;
    this.numberOfRounds = numberOfRounds;
    this.technique = technique;
    this.notes = notes;
  }
}
